Cargo.toml

```diff
@@ -8,8 +8,8 @@ members = [
     "parley_core",
     "parley_dev",
     "parley_draw",
-    "unicode_data",
-    "unicode_data_gen",
+    "parley_data",
+    "parley_data_gen",
```

Comment:
Please move `parley_data*` to before `parley_dev` so that members at this level are alphabetically ordered and match the file tree